create_deb.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+
+version=$(cat setup.py | sed -rn "s/.*version = '(.*?)'.*/\1/p")
+author=$(cat setup.py | sed -rn 's/.*author = "(.*?)".*/\1/p')
+description=$(cat setup.py | sed -rn "s/.*description = '(.*?)'.*/\1/p" | sed 's/ from Python//')
+
+mkdir -p deb/usr/{bin,lib} deb/DEBIAN
+cp usbrelay deb/usr/bin/
+cp libusbrelay.so deb/usr/lib/
```

Comment:
It would be better to reuse our Makefile for installing the package. If the Makefile is updated in the future, it would not be necessary to update this script. Note that I had to fix the Makefile in #68 so that it work correctly on one of my systems.
```suggestion
mkdir -p deb/DEBIAN
make install DESTDIR=deb
```